server/depot.go

```diff
@@ -98,14 +112,93 @@ func (d fileDepot) Serial() (*big.Int, error) {
 		}
 		return s, nil
 	}
-	data, err := ioutil.ReadFile(name)
+	file, err := os.Open(name)
 	if err != nil {
 		return nil, err
 	}
-	serial := s.SetBytes(data)
+	defer file.Close()
+	r := bufio.NewReader(file)
+	// is there a better way??
+	data, err := r.ReadString('\r')
+	data = strings.TrimSuffix(data ,"\r")
```

Comment:
The bytes package provides equivalent methods for working with bytes. 
https://golang.org/pkg/bytes/

See this excellent blog post about using either package https://medium.com/@benbjohnson/go-walkthrough-bytes-strings-packages-499be9f4b5bd#.mpo4ze804